### PR TITLE
Change: semantic HTML-elements for the pages main sections and corresponding CSS-rules

### PR DESCRIPTION
**Pull request by auge8472**
**Changed HTML**

- changed `div#header` into `header#header`
- changed `div#navigation` into `nav#navigation`
- changed `div#content-main` into `main#content-main`
- changed `div#content-bottom` into `footer#content-bottom` **and** separated it from `main`

**Changed CSS**

- removed the background image from `body` and replaced it by a background colour
- changed selector for `#header` into `body > header` (combination because a page can contain several elements `header` (i.e. further `header` for blog articles))
- changed selector for `#navigation` into `body > nav` (combination because a page can contain more than one elements `nav`)
- changed selector `#content-main` into `main` ("There can be only one")
    - centered the link to older blog articles below the blog article section without the use of `float` because `float` extracts the element from the page element flow
- changed selector `#content-bottom` into `body > footer` (combination because a page can contain several elements `footer` (i.e. further `footer` for blog articles))
    - because of the separation of the footer from the main block the footer has now a separate background with rounded corners and box shadow similar to `main`
- kept the IDs because I can't ensure, that they are not addressed from elsewhere (i.e. in CSS-selectors)

### CODE REVIEW

**Comment by auge8472** [comment]
As addition a screenshot from the draft for the footer.

openttd-page-footer

**Comment by auge8472** [comment]
Is there anyone? Everybody in the Easter holidays? ;-)

I hoped to get a reaction to satisfy myself to be on the right way. Same for #85.

File: static/css/page.css [hunk 1]
```diff
@@ -68,6 +68,6 @@
 }
 
 .right {
-	float: right;
 	margin-bottom: 20px;
+	text-align: center;
```
Comment (LordAro) [inline static/css/page.css]: I feel like this change might be confusing, given the name of the class. The class is only used in 2 places - rename?

Comment (auge8472) [reply static/css/page.css]: Seems to be **very** plausible to change the class name. :-)

**Comment by auge8472** [comment]
> Just the one minor nitpick. I feel like I'd prefer it if the "unnecessary" ids were removed as well - what did you mean by "I can't ensure, that they are not addressed from elsewhere" ? Seems like something a text search can deal with :)

I'll search the sources. If I'll find something I can't comprehend I'll aask here. Otherwise I'll add a commit to remove the ids.

**Comment by TrueBrain** [comment]
Ping @LordAro : three tickets with near identical suggested changes, but no activity in a long time. What do you want to do with the pull requests?

**Comment by auge8472** [comment]
After pushing the task more than a half year along, I came into trouble because of the many changes in the meantime. Because of merging of these changes into my branch it came from three to around 20 commits. I tried to squash this into one commit but that would reintroduce all your changes a second time. So I decided to close this PR and opened a new one (#113) with only my own changes (cherry-picked and squashed).

**Pull request closed without merging**